syfertext/pointers/doc_pointer.py

```diff
@@ -28,14 +28,10 @@ def __init__(
             description=description,
         )
 
-    def getEncryptedVector(self, *workers, crypto_provider=None, requires_grad=True):
+    def get_vector(self, *workers, crypto_provider=None, requires_grad=True):
         """
-           Create one big vector composed of the concatenated Token vectors included in the
-           Doc. The returned vector is SMPC-encrypted.
-
-           TODO: This method should probably be removed. It served for a prototype test,
-                 but concatenating all token vectors of the Doc into one big vector
-                 might not be really useful for practical usecases.
+           Get the mean of the vectors of each Token in this documents.
+           Note: The token vectors are SMPC-encrypted.
         """
 
```

Comment:
Please add a complete docstring for args and return values too

Reply:
done